src/test/java/org/worshipsongs/importer/SongParserTest.java

```diff
@@ -93,4 +135,97 @@ public void testParseVerseOrder()
                 "verseOrder=V1 C1 O1 V2 V3\n" +
                 "foofoo"));
     }
+
+    @Test
+    public void testParseLyrics()
+    {
+        assertEquals(lyrics, parser.parseLyrics(lyrics +
+                "\ntitle="));
+        assertEquals(lyrics, parser.parseLyrics(lyrics +
+                "\nauthor="));
+        assertEquals(lyrics, parser.parseLyrics(lyrics));
+        assertEquals("", parser.parseLyrics(""));
+    }
+
+    @Test
+    public void testParseSearchLyrics()
+    {
+        assertEquals("", parser.parseSearchLyrics(""));
+        assertEquals(searchLyrics, parser.parseSearchLyrics(lyrics));
+    }
+
+    @Test
+    public void testGetXmlLyrics()
+    {
+        String expectedLyrics = "";
+        ClassLoader classLoader = getClass().getClassLoader();
+        try {
+            expectedLyrics = IOUtils.toString(classLoader.getResourceAsStream("sample_lyrics.xml"));
```

Comment:
Consider naming the file after the song, for eg: "lord-i-lift-your-name-on-high.xml". Later you may have to write additional tests for different songs.